api/Document.json

```diff
@@ -8629,7 +8629,7 @@
             },
             "edge": [
               {
-                "version_added": "79"
+                "version_added": "12"
```

Comment:
@foolip Yeah, but they were added in version 12 under the alternative name `mspointerdown` (see the section below). In version 79 when Edge when to Blink everything is compatible - ie the alternative_name is no longer correct. So what I did was add a separate entry.

Now you have changed this to 12, we have a confusing story. Did you miss what I was doing?

That seems "correct"

Reply:
Edge wasn't mentioned in the description so I assumed this was a mistake. There are lots of entries that looks something like this:
```
            "edge": [
              {
                "version_added": "12"
              },
              {
                "version_added": "12",
                "version_removed": "79",
                "alternative_name": "onmspointerleave"
              }
            ],
```

That just means both the prefixed and unprefixed variants were supported at the same time. I haven't checked if it's correct, but it's a common situation to have such an overlap.

Reply:
Thanks @foolip. So basically the first case indicates that it is supported for all versions, and the second indicates that from 12 > 79 there was also an alternative name? Makes sense thank you.